init.fish

```diff
@@ -0,0 +1,30 @@
+# SYNOPSIS
+#   exenv [options]
+#
+if not type --quiet "exenv"
+  echo "WARNING: you loaded the fish-shell plugin for exenv, but exenv is not installed."
+end
+
+if status --is-interactive
+  if exenv init - | grep --quiet "function"
+    source (exenv init - | psub)
+  else
+
+    if not set -q EXENV_ROOT
+      set -x EXENV_ROOT "$HOME/.exenv"
+    end
+
+    set PATH "$EXENV_ROOT/shims" $PATH
+
+    function ndenv -d "Elixir version manager"
```

Comment:
@bobthecow indeed I can't move this code to the `functions` folder, because it's all meant to run when the shell is loading.